src/utils/moralis.ts

```diff
@@ -0,0 +1,41 @@
+import Moralis from 'moralis'
```

Comment:
Is making this "indexer.ts" or similar a better idea than vendorname.ts in the event we change vendors? Weak preference for purpose as opposed to implementation.

Reply:
It's rather make it explicit since api will likely depend on the provider somewhat e.g. we need to pass in the current balance where in the xusd-analytics/yield it has the balance available - also sort of follows the convention in use: analytics.js, sentry.js 🤷